packages/firestore/test/integration/api/smoke.test.ts

```diff
@@ -102,20 +101,22 @@ apiDescribe('Smoke Test', persistence => {
   });
 
   asyncIt('will fire value events for empty collections', () => {
-    return integrationHelpers.withTestDb(persistence, db => {
-      const collection = db.collection('empty-collection');
-
-      const accum = new EventsAccumulator<firestore.QuerySnapshot>();
-      const unlisten = collection.onSnapshot(accum.storeEvent);
-      return accum
-        .awaitEvent()
-        .then(querySnap => {
-          expect(querySnap.empty).to.equal(true);
-          expect(querySnap.size).to.equal(0);
-          expect(querySnap.docs.length).to.equal(0);
-        })
-        .then(() => unlisten(), () => unlisten());
-    });
+    return integrationHelpers.withTestCollection(
+      persistence,
+      {},
+      collection => {
+        const accum = new EventsAccumulator<firestore.QuerySnapshot>();
+        const unlisten = collection.onSnapshot(accum.storeEvent);
+        return accum
+          .awaitEvent()
+          .then(querySnap => {
+            expect(querySnap.empty).to.equal(true);
+            expect(querySnap.size).to.equal(0);
+            expect(querySnap.docs.length).to.equal(0);
+          })
+          .then(() => unlisten(), () => unlisten());
```

Comment:
Doesn't this swallow test failures?

Reply:
Sorta'.  I think expect() will store any failures out-of-band and the test will still fail, but if we accidentally threw an exception then it would probably get swallowed.  I removed the error handling case; if the test fails I don't care if it cleans up after itself. :-)

[to be clear, I didn't touch this code originally; it just got reformatted and github can't diff whitespace]